ractor_actors/examples/tcp.rs

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) Sean Lawlor
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree.
+
+extern crate ractor_actors;
+
+use chrono::{DateTime, Utc};
+use ractor::{Actor, ActorProcessingErr, ActorRef};
+use ractor_actors::net::tcp::listener::*;
+use ractor_actors::net::tcp::session::*;
+use ractor_actors::net::tcp::*;
+use ractor_actors::watchdog;
+use ractor_actors::watchdog::TimeoutStrategy;
+use std::error::Error;
+use std::str::FromStr;
+use std::time::SystemTime;
+
+struct MyServer;
+
+struct MyServerState {}
```

Comment:
if it's an empty struct, please don't assign struct args to it. You just can do `struct MyServerState;` and it'll be compiled down to a nothing. 

However if the state is actually empty, the unit struct (`()`) is more idiomatic.